### PR TITLE
Add email instructions and ledgersmb.yaml configuration

### PR DESCRIPTION
**Pull request by neilt**
Stub in configuration for ledgersmb.yaml, which needs to be reviewed. There are a lot of TODOs.

### CODE REVIEW

File: part-configuration.tex [hunk 1]
```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
...
+\subsubsection{\texttt{output\_formatter}}
+@@@TODO  I really don't have a good way to format lists explanations.  See below at \texttt{paths:config:workflows} for an example. Guidance welcome.
+
+\subsubsection{\texttt{paths}}
+
+This section configures the various paths used by LedgerSMB. The configuration parameters are:
+
+\begin{description}
+    \item {\texttt{config:locale}} – Path to locale files. Defaults to \texttt{locale/po}.
+    \item {\texttt{config:sql}} – @@@TODO
```
Comment (ehuelsmann) [inline part-configuration.tex:128]: Where the schema definition files live.

File: part-configuration.tex [hunk 2]
```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
...
+    \item {\texttt{config:sql\_data}} – @@@TODO
```
Comment (ehuelsmann) [inline part-configuration.tex:129]: Where the reference and masterdata files live (i.e. the files with the names of the countries).

File: part-configuration.tex [hunk 3]
```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
...
+    \item {\texttt{config:templates}} – Templates base directory. Typically set to \texttt{templates}.
+    \item {\texttt{config:UI}} – @@@TODO
```
Comment (ehuelsmann) [inline part-configuration.tex:131]: Where the HTML UI files are stored. (Normally './UI/')

File: part-configuration.tex [hunk 4]
```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
...
+    \item {\texttt{config:UI\_cache}} – @@@TODO
```
Comment (ehuelsmann) [inline part-configuration.tex:132]: Where parsed UI templates are cached (so they don't need to be parsed and translated to Perl code again)

**Review by ehuelsmann — APPROVED** [review]
Here's some extra comments/completion. In general: thanks!

File: part-configuration.tex [hunk 5]
```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
...
+    
+\item{ LedgerSMB::Mailer::TransportSMTP} - This email \index{email} is sent using a remove SMTP server.  The options are:
+    \begin{description}
+        \item{\texttt{transport:\$class}} – The required transport to use.
+        \item {\texttt{transport:host}} – The required host name of the smtp \index{ SMTP} server.
+        \item {\texttt{transport:port}} – The required port number of the smtp server. Note this might vary depending on whether TLS or SSL is used.
+        \item {\texttt{sasl\_username:\$class}} – The required smtp server authentication method
+        \item {\texttt{sasl\_username:mechanism}} – @@@TODO What options are available here?
+        \item {\texttt{sasl\_username:callback:user}} – The required SMTP user name.  @@@TODO I assume 'the-user' is just a place holder.
+        \item {\texttt{sasl\_username:callback:pass}} – The required SMTP password.  @@@TODO what does 'SECURITY-FIRST' mean?  Is this just a place holder or does it have meaning to pick up the password from some other place?
```
[the remark this answers is not in the document]
Comment (neilt) [reply part-configuration.tex:115]: Fixed

File: part-configuration.tex [hunk 6]
```diff
@@ -56,17 +56,103 @@ \section{LedgerSMB version numbers}
...
+    \end{description}
+    
+\item{ LedgerSMB::Mailer::TransportSMTP} - This email \index{email} is sent using a remove SMTP server.  The options are:
+    \begin{description}
+        \item{\texttt{transport:\$class}} – The required transport to use.
+        \item {\texttt{transport:host}} – The required host name of the smtp \index{ SMTP} server.
+        \item {\texttt{transport:port}} – The required port number of the smtp server. Note this might vary depending on whether TLS or SSL is used.
+        \item {\texttt{sasl\_username:\$class}} – The required smtp server authentication method
+        \item {\texttt{sasl\_username:mechanism}} – @@@TODO What options are available here?
+        \item {\texttt{sasl\_username:callback:user}} – The required SMTP user name.  @@@TODO I assume 'the-user' is just a place holder.
```
[the remark this answers is not in the document]
Comment (neilt) [reply part-configuration.tex:114]: Fixed

File: part-configuration.tex (hunk 1 above)
Comment (neilt) [reply part-configuration.tex:132]: Added

File: part-configuration.tex (hunk 2 above)
Comment (neilt) [reply part-configuration.tex:133]: Added

File: part-configuration.tex (hunk 3 above)
Comment (neilt) [reply part-configuration.tex:135]: Added

File: part-configuration.tex (hunk 4 above)
Comment (neilt) [reply part-configuration.tex:136]: Added

**Pull request merged**